operator/Dockerfile.ocp

```diff
@@ -0,0 +1 @@
+build/Dockerfile
```

Comment:
My thought here is to make the request to the ART team after this PR is merged since the other file is remaining in the short term.